dagshub/streaming/mount.py

```diff
@@ -97,6 +142,15 @@ def read(self, path, size, offset, fh):
             return os.read(fh, size)
 
     def readdir(self, path, fh):
+        """
+        List the contents of a directory.
```

Comment:
Does this LIST or READ directory?

Reply:
From the function,it lookslike it lists the contents of a directory and returns it

Reply:
I'd suggest to not document the mount/filesystem classes this thoroughly. (sorry im only saying this right now)
The only points of interaction for them are `mount()` and `install_hooks()`, and everything else is there to make it work.